README.md

```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+## Who am I? 
+My name is Liam Cohen. I am currently a **rising third year** in the College
+majoring in *Latin American Studies and Political Science*. I highly recommend 
+that you check out the 
+[Center for Latin American Studies](https://clas.uchicago.edu/). 
```

Comment:
Cool use of the link function!

Reply:
Great job formatting the alt text in such a way as to not have it read "TEXT HERE" or whatever the automatic thing is. It took me a hot minute to figure out how to make it look like you did. Good job!